node/monitor.js

```diff
@@ -165,21 +167,98 @@ OpKindMonitor.prototype.summary = function summary() {
     if (self.inCounts.length) {
         self.log('= %s IN COUNTS: %j',
                     self.options.desc,
-                    self.inCounts.reduce(sumCounts));
+                    self.inCounts.reduce(sumCounts, {}));
```

Comment:
Should probably pop these uses of array methods regardless of how cold these paths are.